os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -431,6 +435,18 @@ void up_assert(const uint8_t *filename, int lineno)
 
 	board_led_on(LED_ASSERTION);
 
+#ifdef CONFIG_CPP_HAVE_VARARGS
+	va_list valist;
+	const char *assertInfo;
+
+	/* Extract the extra debug arguments if any */
+	va_start(valist, lineno);
+	for (int i = 0; i < 10; i++) {
```

Comment:
Is there any special reason to check 10 times?
If it is, how about declaring the value with proper name?

Reply:
Not needed. Updated the code.